experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -798,6 +798,68 @@ describe("SharedTree", () => {
 			unsubscribe1();
 			unsubscribe2();
 		});
+
+		/**
+		 * the collab window includes all sequenced edits after the minimum sequence number
+		 * these tests test that undoing edits beind the minimum sequence number works
+		 */
+		describe("out of collab window", () => {
+			for (const opNumber of [20, 40, 100]) {
```

Comment:
I don't think we get much value out of testing different numbers here so long as we can guarantee that the change we're undoing is out of the collab window. Which begs the question: how can we be sure that it is indeed out of the collab window? It would be good to have a way to do that, otherwise we can't be sure the test is really testing what we want. Does `provider.processMessages()` always bring the minSeqNumber up? If not, how do we check programmatically what the minSeqNumber is? You might have to ask around.